### PR TITLE
`cca create` and `cca check` sub-commands

### PR DESCRIPTION
**Pull request by thomas-fossati**
Partially addresses #15

### CODE REVIEW

File: cmd/cca/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Claims Template
+
+```json
+{
+  "cca-platform-token": {
+    // CCA Platform claims
```
Comment (yogeshbdeshpande) [inline cmd/cca/README.md:6]: Does the comments allowed like that in the JSON templates. I thing, we do not need them here anyway!

File: cmd/cca/check.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+Check a CCA attestation token contained in my.cbor using the public PAK in
+es256.jwk and save the embedded claims to claims.json:
+
+	evcli cca check --token=my.cbor --key=es256.jwk --claims=claims.json
+
+Or, equivalently:
+
+	evcli cca check -t my.cbor -k es256.jwt -c claims.json
+
+check a CCA attestation token contained in te.cbor using the public PAK in
```
Comment (yogeshbdeshpande) [inline cmd/cca/check.go:39]:
```suggestion
check a CCA attestation token contained in te.cbor using the public IAK in
```

**Review by yogeshbdeshpande — APPROVED** [review]
Left few minor comments about the name IAK we use consistently in Spec for CCA Will be good to follow that! Apart from that LGTM!

File: cmd/cca/check.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+var checkCmd = NewCheckCmd(common.Fs)
+
+func NewCheckCmd(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "check",
+		Short: "do the syntactic and cryptographic signature checks over a CCA attestation token",
+		Long: `Run the syntactic and cryptographic signature checks over the
+supplied CCA attestation token.
+
+Check a CCA attestation token contained in my.cbor using the public PAK in
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply cmd/cca/check.go:30]: the naming (at least in the "CCA security model" document) is CPAK: CCA Platform Attestation Key

[the remark this answers is not in the document]
Comment (thomas-fossati) [reply cmd/cca/check.go:30]: I've added a point in #15 to take care of naming alignment

File: cmd/cca/check.go (hunk 2 above)
Comment (thomas-fossati) [reply cmd/cca/check.go:39]: ditto

File: cmd/cca/README.md (hunk 1 above)
Comment (thomas-fossati) [reply cmd/cca/README.md:6]: This is just a very sketchy WIP.  There's a specific point in #15 to provide documentation.

**Pull request merged**